pkg/ansible/proxy/cache_response.go

```diff
@@ -79,7 +74,12 @@ func (c *cacheResponseHandler) ServeHTTP(w http.ResponseWriter, req *http.Reques
 		k, err := getGVKFromRequestInfo(r, c.restMapper)
 		if err != nil {
 			// break here in case resource doesn't exist in cache
-			log.Info("Cache miss, can not find in rest mapper")
+			log.Info("Cache miss, can not find in rest mapper", "request", r, "gvk", k)
```

Comment:
is `r` the request or the resource? Also, are we sure we should be logging that variable, or should we just log the gvk?